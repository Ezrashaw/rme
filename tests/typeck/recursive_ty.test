fn id(x) = x(x);
id(id);