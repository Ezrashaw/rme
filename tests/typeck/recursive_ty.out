[1;3;31merror[0m: infinite sized type
